src/main/java/PrintTask.java

```diff
@@ -0,0 +1,51 @@
+public class PrintTask {
+    public static void list(Task[] tasks) {
+        int index = 1;
+        PrintText.print("Here are the tasks in your list:");
+        for (Task task : tasks) {
+            String indexPrinted = index + ".";
+            char type = task.getTypeIcon();
+            String typeMark = "[" + type + "]";
+            String statusMark = "[" + task.getStatusIcon() + "] ";
+            PrintText.print(indexPrinted + typeMark + statusMark + task.description);
+            index++;
+        }
+    }
+
+    public static void specialTask(Task newTask, int taskNum) {
+        String intro = "Got it. I've added this task: \n";
+        char type = newTask.getTypeIcon();
+        String taskDisplayed;
+        String[] words = newTask.description.split(" ");
+        switch(words[0]) {
```

Comment:
this switch is similar to the switch in addTask class, maybe you can use abstraction principle to combine them into one logic class to improve the readability :)